src/components/OfferDialog.jsx

```diff
@@ -28,10 +29,14 @@ const OfferDialog = ({
   quickBid,
   mylisting
 }) => {
+  console.log("opened:",minAmount.toString());
   const [open, setOpen] = useState(false);
-  const [input, setInput] = useState(minAmount);
+  let [input, setInput] = useState(minAmount);
```

Comment:
hmm I don't think we should ever use `let` for states. If we need it somewhere, then the code isn't correct or isn't following best practice, and may have unpredicatable behavior.  I think we can do `const [input, setInput] = useState(ethers.utils.parseEther(minAmount.toString()))`